src/main/java/de/hysky/skyblocker/config/configs/UIAndVisualsConfig.java

```diff
@@ -128,9 +129,15 @@ public static class TabHudConf {
         @SerialEntry
         public boolean enableHudBackground = true;
 
+        @SerialEntry
+        public boolean effectsFromFooter = false;
+
         @SerialEntry
         public boolean plainPlayerNames = false;
 
```

Comment:
This option and `nameSorting` seems to have been removed? It's fine to keep it here ig, just checking.

Reply:
I deprecated them.